### DISCUSSION TITLE
Decode the hidden quote 🤐

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1246] [Galaxy Trivia] [answered]
The two parts (also known as rails) of this cipher are separated by the pipe (|) symbol. Each rail contains alternating letters in the hidden message. The asterisk (*) symbol represents a space. You have finished the puzzle when you have decoded the full message.

```
TEWA*A*EE*OGV.F
RIEESI*H*TRBT*F
TESRN.|H*EKCNNV
RFRIE*OGVNS*STE
ATIUEO*H*TOG
```

**Bonus:** Name the author.

**Comment by peterbaccaro** [comment, marked as answer]
The hidden quote is “**The weak can never forgive. Forgiveness is the attribute of the strong.**” and is attributed to **Mahatma Gandhi**.

**Comment by KR-Ravindra** [comment]
THE BEST THINGS IN LIFE ARE FREE 

Buddy DeSylva and Lew Brown (lyrics) and Ray Henderson (music) for the 1927

**Reply by ParanoidUser** [reply]
Interesting to know where these lyrics were hidden. 😀🔍
